packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.constants.js

```diff
@@ -31,13 +31,18 @@ export const CATEGORIES = [
     title: 'IOPS',
     pattern: /ovh\.iops/,
   },
+  {
+    category: 'baremetal',
+    title: 'Bare Metal',
+    pattern: /baremetal/,
+  },
 ];
 
 export const FLEX_TYPE = /flex$/;
 
 export const LEGACY_FLAVORS = /eg|sp|hg|vps-ssd/;
 
-export const SSD_DISK_TYPES = [/ssd/, /nvme/, /iops/];
+export const SSD_DISK_TYPES = [/ssd/, /nvme/, /iops/, /baremetal/];
 
 export const CPU_FREQUENCY = {
   'ovh.vps-ssd': 2.4,
```

Comment:
This is just a regex to determine whether the disk type is `ssd` or `ceph`. Do you think this should come from the APIs?

Reply:
Yes I think so as it is deep product logic

Reply:
The current APIs do not support this. Hence I have created a new bug to track this: MANAGER-4264, so that we are not blocked in releasing Baremetal support by this issue.